### PR TITLE
media-gfx/enblend: Version bump to 4.2

### PR DESCRIPTION
**Pull request by P424D0X**

### CODE REVIEW

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: `EAPI=6` please

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+src_compile() {
+	# forcing -j1 as every parallel compilation process needs about 1 GB RAM.
+	cmake-utils_src_compile -j1
+}
+
+src_install() {
+	local DOCS=( AUTHORS ChangeLog NEWS README )
+	cmake-utils_src_install
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: in `EAPI=6` you should be able to get rid of `src_install` completely, as the `DOCS` are part of the default set in `einstalldocs`

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_CXX_FLAGS_RELEASE=""
+		$(cmake-utils_use_enable debug DMALLOC)
+		$(cmake-utils_use doc DOC)
+		$(cmake-utils_use_enable image-cache IMAGECACHE)
+		$(cmake-utils_use_enable openmp)
+		$(cmake-utils_use_enable gpu)
+	)
+	CMAKE_BUILD_TYPE="Release"
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: Can you try unscrewing this? In general, Gentoo only officially supports building with the default `CMAKE_BUILD_TYPE="Gentoo"`

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_CXX_FLAGS_RELEASE=""
+		$(cmake-utils_use_enable debug DMALLOC)
+		$(cmake-utils_use doc DOC)
+		$(cmake-utils_use_enable image-cache IMAGECACHE)
+		$(cmake-utils_use_enable openmp)
+		$(cmake-utils_use_enable gpu)
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: you'll have to convert these for `EAPI=6`

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+	)
+
+src_prepare() {
+	sed -i -e "/CXX_FLAGS/s:-O3::g" CMakeLists.txt || die
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_CXX_FLAGS_RELEASE=""
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: try and patch the build system to get rid of this awfulness

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+		"${FILESDIR}/${PN}-4.1.3-vigra_check.patch"
+		"${FILESDIR}/${PN}-4.2-add-missing-cmakelists.patch"
+	)
+
+src_prepare() {
+	sed -i -e "/CXX_FLAGS/s:-O3::g" CMakeLists.txt || die
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: can you integrate this into one of the patches and then completely nuke `src_prepare`?

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+		sci-visualization/gnuplot[gd]
+		virtual/latex-base
+	)"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+		"${FILESDIR}/${PN}-4.1.3-vigra_check.patch"
+		"${FILESDIR}/${PN}-4.2-add-missing-cmakelists.patch"
+	)
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: too much indentation

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+LICENSE="GPL-2 VIGRA"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
+IUSE="debug doc gpu image-cache openmp"
+
+REQUIRED_USE="openmp? ( !image-cache )"
+
+RDEPEND="
+	>=dev-libs/boost-1.31.0:=
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: Ancient version, instead have you checked whether this package builds against the latest **boost-1.62**? See also https://bugs.gentoo.org/show_bug.cgi?id=595630

File: media-gfx/enblend/enblend-4.2.ebuild [hunk 9]
```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils cmake-utils
```
Comment (SoapGentoo) [inline media-gfx/enblend/enblend-4.2.ebuild]: what do you `need` eutils for?

**Comment by gktrk** [comment]
@gentoo/graphics

**Comment by SoapGentoo** [comment]
@P424D0X Please perform requested edits for your next PR, a more recent version is in the tree now.

**Pull request closed without merging**